tests/boulder-integration.sh

```diff
@@ -217,3 +220,5 @@ if ${CI:-false} || type nginx;
 then
     . ./certbot-nginx/tests/boulder-integration.sh
 fi
+
+coverage report --fail-under 52 -m
```

Comment:
Oh yeah. It was 52 when I tested locally but it didn't run the nginx tests because I didn't have nginx installed. I'll update this number when I see where the coverage is at including #4855.